addons/web/static/tests/views/form_tests.js

```diff
@@ -4823,6 +4823,51 @@ QUnit.module('Views', {
         form.destroy();
     });
 
+    QUnit.test('no focus set on form when closing many2one modal if lastActivatedFieldIndex is not set', function (assert) {
```

Comment:
this test actually does not verify that the scroll position hasn't moved. Please add a test that reproduce the bug described in the task exactly, and assert that the bug is fixed.

Reply:
@VincentSchippefilt Bug is not about scroll position, actually form is scrolled because focus goes to first field in form, so issues is first field activated, so I tested that we don't have focus on form when dialog is closed if we don't have ```lastActivatedFieldIndex``` I will still check if we can improve test

Reply:
@VincentSchippefilt I improved test case bit, removed unnecessary asserts which is not needed in our case, if you still want to test scroll then I need design form such a way that we have scroll but issue is not about scrolling, do you want to test scroll position then let me know

Reply:
I think the visible issue was the scrolling (on top of the invisible issue of the focus incorrectly reset). I would rather spend some time designing a test that validates both, to make sure we don't break the scroll in a new way in the futur.
Thanks in advance

Reply:
@VincentSchippefilt added fix height for ActionManager element to have scroll and added asserts to test scrollTop, would you check now?